flask_monitoringdashboard/frontend/js/controllers/configuration.js

```diff
@@ -0,0 +1,98 @@
+export function ConfigurationController($scope, $http, menuService, endpointService, modalService) {
+    endpointService.reset();
+    menuService.reset('configuration');
+    modalService.setConfirm('delete', () => deleteUser($scope.user));
+    modalService.setConfirm('edit', () => editUser($scope.user));
+    modalService.setConfirm('create', createUser);
+
+    $scope.details = {};
+    $scope.config = {};
+    $scope.error = {};
+    $scope.userData = [];
+
+    $http.get('api/users').then(function (response) {
+        $scope.userData = response.data;
+    });
+
+    $http.get('api/deploy_details').then(function (response) {
+        $scope.details = response.data;
+    });
+    $http.get('api/deploy_config').then(function (response) {
+        $scope.config = response.data;
+    });
+
+    $scope.openModal = function(name, user){
+        $scope.user = user;
+        $(`#${name}Modal`).modal();
+    }
+
+    function fetchUsers(){
+        $http.get('api/users').then(function (response) {
+            $scope.userData = response.data;  // reload user data
+        });
+    }
+
+    function createUser(){
+        $http.post(
+            'api/user/create',
+            $.param({
+                'username': $('#create-username')[0].value,
+                'password': $('#create-pwd')[0].value,
+                'password2': $('#create-pwd2')[0].value,
+                'is_admin': $('#create-admin')[0].checked,
+            }),
+            {
+                headers: {
+                    'Content-Type': 'application/x-www-form-urlencoded;charset=utf-8;'
```

Comment:
the most minor of the minor observations: shouldn't this be a constant somewhere?